src/KernelFunctions.jl

```diff
@@ -35,7 +35,7 @@ export spectral_mixture_kernel, spectral_mixture_product_kernel
 
 export ColVecs, RowVecs
 
-export MOInput
+export IsotopicByFeatures, IsotopicByOutputs
 export IndependentMOKernel,
     LatentFactorMOKernel, IntrinsicCoregionMOKernel, LinearMixingModelKernel
```

Comment:
```suggestion
    LatentFactorMOKernel, IntrinsicCoregionMOKernel, LinearMixingMOKernel
```
(can be a separate PR)

Reply:
Same as above question regarding changing the name. There was a [thread](https://github.com/JuliaGaussianProcesses/KernelFunctions.jl/pull/304#discussion_r655369071) about the naming of this `MOKernel` in the PR that added it.

It is called the Linear Mixing Model or LMM so I think either one of those needs to be in the name. Either way we can maybe discuss that in a separate issue :)

Reply:
It's also called Intrinsic Coregionalization Model and [Semiparametric] Latent Factor Model - the "Model" doesn't add much, but distinguishing between multi-output kernels and single-output kernels seems useful to me:)

Reply:
fwiw I'd be against the abbreviations...

Reply:
I agree adding the `MO` could be beneficial. However I believe it is important to maintain `Linear Mixing Model` in its entirety as this is the name that it was assigned by those who developed it and will be using it (of course not using it exclusively). I could be wrong though, @willtebbutt @nickrobinson251 may also have thoughts

Reply:
Re abbreviations I think that would just potentially be an alias added (discussed in the last PR, dropped for now)

Reply:
It's only a *model* once you do something with the kernel, so I don't think it makes sense to include "Model" as part of the name. It *is* a multi-output kernel that is created by linearly mixing of kernel matrices.

Reply:
Agreed -- for example, I would be completely fine with `LinearMixingKernel` or something like that.

Reply:
Well, it comes down to whether you think saving the 10% extra characters from "LinearMixingMOKernel" vs "LinearMixingKernel" offsets the corresponding loss in explicitness. Users of the package can always use their own aliases to make their own code shorter. But KernelFunctions only exposes a single name space that has a lot of entities already, so making it easier for people to distinguish between single-output and multi-output kernels is valuable. (In my experience, in practice you often want to do quite different things for multi-output *models* so easily knowing whether a kernel is multi-output or not is useful.)

Reply:
I would argue that you can already distinguish them since (currently) all multi-output kernels are a subtype of `MOKernel` (which maybe should rather be called `MultiOutputKernel`).

Reply:
I'd be happier with that if there was a SingleOutputKernel as well ... I'm not sure if MOKernel being a subtype of Kernel is really a good idea...

Reply:
Is it not okay to view a multi-output kernel as a special case of a regular kernel?

Reply:
Discussion bumped to an [issue](https://github.com/JuliaGaussianProcesses/KernelFunctions.jl/issues/332)
:)

Reply:
Discussion moved to a new [issue](https://github.com/JuliaGaussianProcesses/KernelFunctions.jl/issues/332)